samples/bluetooth/iso_broadcast/src/main.c

```diff
@@ -175,11 +179,22 @@ int main(void)
 		printk("BIG create complete chan %u.\n", chan);
 	}
 
+	uint32_t skip = 3U;
```

Comment:
What is the purpose of this `skip` variable?

Reply:
This a separate commit to test that one stream starting later in time does synchronize to the same timestamp across all streams in the group.